card/src/main/java/com/adyen/checkout/card/CardView.kt

```diff
@@ -577,4 +609,18 @@ class CardView @JvmOverloads constructor(context: Context, attrs: AttributeSet?
             else -> null
         }
     }
+
+    fun setSupportedCardsList(cards: List<CardType>) {
```

Comment:
I think this breaks a bit the idea if this change. Afaik, we want the list to show up in the standalone component as well, so the `supportedCardsList` should be sent by the component in the `CardInputData` instead of someone having to inject it into the view manually. This works well for Drop-in only flow, but I'm not sure if that's the intent here.

Reply:
Ah okay, let me try to get it working with `CardInputData `

Reply:
While you're at it make sure to test the stored card flow as well, not sure how it should look after this change.